js/common.js

```diff
@@ -126,7 +126,7 @@ LayerScope.TextureNode = function(ptexture, texID, layerRef, newContent) {
 
     if (!!ptexture.mask.mMaskTransform) {
       this.mMaskTransform = {
-        m: [ele for (ele of ptexture.mask.mMaskTransform.m)]
+        m: Array.from(ptexture.mask.mMaskTransform.m.map)
```

Comment:
This confuses me because you use "m.map" as the parameter, not the original array object, "m". Is there any reason for it? Thanks.